### ISSUE TITLE
Questions about cal_sc_loss?

### ISSUE THREAD

**Issue by WenxiongLiao** [laihuiyuan/Pre-trained-formality-transfer#6]
Hey, can you explain cal_sc_loss?  What is the calculation process of loss? The y' generated by the decoder has been discretized. Why can we calculate its gradient to fine-tuning the language model?
